tests/Unit/PointwiseFunctions/AnalyticData/CurvedWaveEquation/Test_ScalarWaveGr.cpp

```diff
@@ -383,6 +473,26 @@ SPECTRE_TEST_CASE("Unit.AnalyticData.CurvedWaveEquation.ScalarWaveGr",
     test_kerr(curved_wave_data, bh_solution, flat_wave_solution);
     test_kerr_schild_vars(curved_wave_data, bh_solution);
   }
+  {  // test with wave profile = plane in flat space in 3D
```

Comment:
[optional] It seems a bit strange to me that all the Kerr variables (mass_spin, bh_solution, and especially `background_tag` which is probably the easiest to use accidentally because it doesn't sound like a Kerr thing) are still in scope for this new flatspace test.  And none of those variables are used for this flatspace test.

Perhaps (as in many other tests) this test can be refactored into two functions: the first function is the test before this PR (which includes all of the blocks above), and the second function is this new test.  Then `SPECTRE_TEST_CASE("Unit.AnalyticData.CurvedWaveEquation.ScalarWaveGr",...` can be very short and simple, i.e. it just calls two functions.

Reply:
Thats fair, I've pushed a fixup splitting this test.

Reply:
@markscheel - can I squash this fix?

Reply:
I'm not a reviewer on this PR, so I let @wthrowe decide here, but we've been keeping the number of test cases low in favor of a single test case that calls into a bunch of functions. This is to reduce overhead from test-startup times. My suggestion is to change both test cases you have here into functions (comes down to a two-line change), and call them from a single test case. If you don't want to bother with that, I think you can also just drop the fixup commit. If you agree to make the change, just squash it into the existing fixup. Then we can get this merged once people in the US wake up :)

Reply:
Sure, I moved the two tests into functions and now call them from a single test case. This now satisfies both Mark's and your comments.

I've squashed this change with older fixups, and the PR is good to go now.